README.md

```diff
@@ -53,9 +54,48 @@ This will display the metrics in the terminal.
 
 ### Custom config file
 
```

Comment:
We could add that if they want to track only chainflip network and node/s they can disable btc, eth and polkadot, I guess many people don't care about tracking everything

Reply:
Good idea